syft/source/directory_resolver.go

```diff
@@ -7,11 +7,12 @@ import (
 	"path"
 	"path/filepath"
 
-	"github.com/anchore/stereoscope/pkg/file"
 	"github.com/anchore/syft/internal/log"
 	"github.com/bmatcuk/doublestar"
 )
 
+var _ Resolver = &DirectoryResolver{}
```

Comment:
Not a necessary change, but just a note that there's also an allocation-free variant of that check that the Go folks recommend: `var _ Resolver = (*DirectoryResolver)(nil)`

Reply:
I always forget this one... good call